examples/js/renderers/CSS3DRenderer.js

```diff
@@ -52,26 +53,21 @@ THREE.CSS3DRenderer = function () {
 	var domElement = document.createElement( 'div' );
 	domElement.style.overflow = 'hidden';
 
-	domElement.style.WebkitTransformStyle = 'preserve-3d';
-	domElement.style.MozTransformStyle = 'preserve-3d';
-	domElement.style.oTransformStyle = 'preserve-3d';
-	domElement.style.transformStyle = 'preserve-3d';
-
 	this.domElement = domElement;
 
 	var cameraElement = document.createElement( 'div' );
 
-	cameraElement.style.position = 'absolute';
-	cameraElement.style.left = '50%';
-	cameraElement.style.top = '50%';
-
 	cameraElement.style.WebkitTransformStyle = 'preserve-3d';
 	cameraElement.style.MozTransformStyle = 'preserve-3d';
-	cameraElement.style.oTransformStyle = 'preserve-3d';
 	cameraElement.style.transformStyle = 'preserve-3d';
 
 	domElement.appendChild( cameraElement );
 
+	// Should we replace to feature detection?
+	// https://github.com/Modernizr/Modernizr/blob/master/feature-detects/css/transformstylepreserve3d.js
+	// So far, we use `document.documentMode` to detect IE
+	var isFlatTransform = !! document.documentMode;
```

Comment:
@mrdoob Thanks for your review❤️

Up to you! Either is okay for me.
It's just for if there was another unknown browser that does not support `preserve-3d`.

IE11's `userAgent` doesn't contain `'MSIE'` actually
IE11: `Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; Touch; rv:11.0) like Gecko`
IE10: `Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; Trident/6.0)`
https://msdn.microsoft.com/library/hh869301(v=vs.85).aspx#code-snippet-4

maybe `var isIE = /Trident/i.test( navigator.userAgent )`?